internal/uvm/virtual_device.go

```diff
@@ -31,6 +37,12 @@ type VPCIDevice struct {
 	refCount uint32
 }
 
+// A vpci bus's instance ID is in the form: "VMBUS\vmbusChannelGUIDFormatted\{vmBusInstanceGUID}"
+func (uvm *UtilityVM) GetAssignedDeviceParentID(vmBusInstanceGUID string) string {
```

Comment:
Throughout this PR there are various terms like "parent ID", "instance ID", "location path", etc. Is there an overview in a comment somewhere? I think that would be good to have.

Reply:
Added a new comment above this function explaining those.